common/worklet-reftest.js

```diff
@@ -1,7 +1,12 @@
-// Imports code into a worklet. E.g.
-//
-// importWorklet(CSS.paintWorklet, {url: 'script.js'});
-// importWorklet(CSS.paintWorklet, '/* javascript string */');
+/**
+ * Imports code into a worklet. E.g.
+ *
+ * importWorklet(CSS.paintWorklet, {url: 'script.js'});
+ * importWorklet(CSS.paintWorklet, '(javascript string)');
+ *
+ * @param {PaintWorklet} worklet
```

Comment:
'PaintWorklet' is wrong here; it will be any worklet (https://developer.mozilla.org/en-US/docs/Web/API/Worklet)

Reply:
```suggestion
 * @param {Worklet} worklet
```